README.md

```diff
@@ -27,6 +27,22 @@ This product distribution is made available under the [Apache 2.0 license](http:
 
 Portions of this product distribution require certain copyright or other notices to be included with the product distribution. These notices may also appear in individual source files. 
 
+Project Requirements
+---
+- Cocoapods
```

Comment:
CocoaPods is not technically a *requirement* for this project, as we commit the pod projects to this repo. It's only needed if a project fork wants to update one or more pods, or modify the project dependencies.